packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -87,6 +87,62 @@ function useTokenFromSearchParams(): string | undefined {
   return tokenFromSearchParams
 }
 
+const getSmartContractTransferError = async ({
+  from,
+  to,
+  l1Provider,
+  l2Provider,
+  isDeposit
+}: {
+  from: string
+  to?: string
+  l1Provider: Provider
+  l2Provider: Provider
+  isDeposit: boolean
+}): Promise<TransferValidationErrors | null> => {
```

Comment:
NIT: Can we call it `CustomDestinationTransferValidationErrors` since these are just smart contract specific?